docs/core/docker/build-container.md

```diff
@@ -122,205 +135,194 @@ Counter: 4
 
 If you pass a number on the command line to the app, it will only count up to that amount and then exit. Try it with `dotnet run -- 5` to count to five.
 
-> [!NOTE]
+> [!IMPORTANT]
 > Any parameters after `--` are not passed to the `dotnet run` command and instead are passed to your application.
 
 ## Publish .NET Core app
 
-Before you add your .NET Core app to the Docker image, publish it. You want to make sure that the container runs the published version of the app when it's started.
-
-From the working folder, enter the *app* folder with the example source code and run the following command:
+Before adding the .NET Core app to the Docker image, first it must be published. It is best to have the container run the published version of the app. To publish the app, execute the following command:
```

Comment:
We're not supposed to use the word "Execute" which, in translation, may not mean run, but mean kill 😄 There are a few places where you've used this word.

https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33512

Reply:
Good catch